src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -24,997 +24,998 @@ namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
 
     internal partial class DiagnosticIncrementalAnalyzer : BaseDiagnosticIncrementalAnalyzer
     {
-        private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
-        private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
+            private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
+            private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
```

Comment:
Did you have any changes? It looks like some methods are being moved\deleted as well (hard to tell since the diff is messed up). Can you fix up this file so that the changes are obvious?